_in_container_build.sh

```diff
@@ -70,35 +60,35 @@ for TARGET in $TARGET_LIST; do
 
     # Actually build
     echo "Building kernel for $TARGET"
-    make -C /app/linux ARCH=${short_arch} CROSS_COMPILE=$(get_cc $TARGET) O=/tmp/build/${TARGET}/ olddefconfig
+    make -C /app/linux/$VERSION ARCH=${short_arch} CROSS_COMPILE=$(get_cc $TARGET) O=/tmp/build/${TARGET}/ olddefconfig
 
     # If updating configs, lint them with kernel first! This sorts, removes default options and duplicates.
-    if $CONFIGONLY; then
+    if $CONFIG_ONLY; then
       echo "Updating $TARGET config in place"
-      make -C /app/linux ARCH=${short_arch} CROSS_COMPILE=$(get_cc $TARGET) O=/tmp/build/${TARGET}/ savedefconfig
+      make -C /app/linux/$VERSION ARCH=${short_arch} CROSS_COMPILE=$(get_cc $TARGET) O=/tmp/build/${TARGET}/ savedefconfig
       cp /tmp/build/${TARGET}/defconfig /app/config.${TARGET}
       echo "Finished update for config.${TARGET}"
     else
-      make -C /app/linux ARCH=${short_arch} CROSS_COMPILE=$(get_cc $TARGET) O=/tmp/build/${TARGET}/ $BUILD_TARGETS -j$(nproc)
+      make -C /app/linux/$VERSION ARCH=${short_arch} CROSS_COMPILE=$(get_cc $TARGET) O=/tmp/build/${TARGET}/ $BUILD_TARGETS -j$(nproc)
+
+      mkdir -p /kernels/$VERSION
 
       # Copy out zImage (if present) and vmlinux (always)
       if [ -f "/tmp/build/${TARGET}/arch/${short_arch}/boot/zImage" ]; then
-          cp "/tmp/build/${TARGET}/arch/${short_arch}/boot/zImage" /kernels/zImage.${TARGET}
+          cp "/tmp/build/${TARGET}/arch/${short_arch}/boot/zImage" /kernels/$VERSION/zImage.${TARGET}
       fi
-      cp "/tmp/build/${TARGET}/vmlinux" /kernels/vmlinux.${TARGET}
+      cp "/tmp/build/${TARGET}/vmlinux" /kernels/$VERSION/vmlinux.${TARGET}
 
       # Generate OSI profile
-      echo "[${TARGET}]" >> /kernels/osi.config
+      echo "[${TARGET}]" >> /kernels/$VERSION/osi.config
```

Comment:
We could merge all the configs into a single file, it's just a `[key]` followed by a bunch of values

Reply:
Are you sure? It looks like the beginning of the file is

```ini
[armel]
name = 4.10.0|#1 Wed Jan 17 17:42:47 UTC 2024|arm
version.a = 4
version.b = 10
version.c = 0
```

I think we would be able to do that if the key had both the arch and version, but I think as-is the fields might be treated as duplicates leading to errors or all but one config being ignored?